lib/straight/blockchain_adapters/chain_com_adapter.rb

```diff
@@ -0,0 +1,79 @@
+module Straight
+  module Blockchain
+
+    class ChainComAdapter < Adapter
+
+      MAINNET_BASE_URL = 'https://api.chain.com/v2/bitcoin'
+      TESTNET_BASE_URL = 'https://api.chain.com/v2/testnet3'
+
+      def self.mainnet_adapter(api_key_id:)
+        new(api_key_id)
+      end
+
+      def self.testnet_adapter(api_key_id:)
+        new(api_key_id, testnet: true)
+      end
+
+      def initialize(api_key_id, options = {})
```

Comment:
No so important, but I think it's better to use named arguments instead of the old-style `options`.